be/src/exec/aggregator.cpp

```diff
@@ -413,6 +414,23 @@ Status Aggregator::prepare(RuntimeState* state, ObjectPool* pool, RuntimeProfile
             Expr* expr = nullptr;
             ExprContext* ctx = nullptr;
             RETURN_IF_ERROR(Expr::create_tree_from_thrift(_pool, desc.nodes, nullptr, &node_idx, &expr, &ctx, state));
+
+            if (state->query_options().__isset.enable_jit && state->query_options().enable_jit) {
+                auto* jit_wrapper = JITWapper::get_instance();
+                if (jit_wrapper->support_jit()) {
+                    const auto* prev_e = expr;
+                    auto status = expr->replace_compilable_exprs(&expr, _pool);
+                    if (!status.ok()) {
+                        LOG(ERROR) << "Could replace compilable exprs.\n" << status.get_error_msg() << "\n";
```

Comment:
if not ok, it continues use non-jit code?

Reply:
For expressions currently supported by JIT, yes.